src/live/customize.rs

```diff
@@ -169,6 +175,28 @@ impl LiveInitrd {
         Ok(())
     }
 
+    pub fn network_nmstate(&mut self, path: &str) -> Result<()> {
+        if !self.features.live_initrd_network {
+            bail!("This OS image does not support customizing network settings.");
+        }
+        let net_state_str = read_to_string(path)?;
```

Comment:
We generally avoid bare `?` operators when calling external libraries.  Let's add `context()` or `with_context()` calls that explain what we were doing at the time of the failure.

Reply:
Done